ui/src/Views/UserInformation/UserInformation.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState, useEffect, useContext } from "react";
+import { postUserInformation } from "./services";
+import { Context } from "../../store/reducer";
+
+const UserInformation = () => {
+  const [distance, setDistance] = useState(0);
+  const [radio, setRadio] = useState("hero");
+  const [geoError, setGeoError] = useState(false);
+  const [coordinates, setCoordinates] = useState<number[]>([]);
+  const [state] = useContext(Context);
+
+  const handleUserInformation = (e: any) => {
+    const { name, phoneNumber, description } = e.target.values;
+
+    if (
+      name.value !== "" &&
+      phoneNumber.value !== "" &&
+      description.value !== "" &&
+      !geoError &&
+      state.userId &&
+      state.idToken
+    ) {
+      postUserInformation(
+        state.userId,
+        state.idToken,
+        name.value,
+        phoneNumber.value,
+        distance,
+        description.value,
+        coordinates,
+        radio
+      );
+    }
+  };
```

Comment:
Skapa en function istallet. Blir lite jobbig att lasa

Reply:
vad menar du?

Reply:
Tanker mig kanske att skicka in hela state, sa loser du ut `userId` och `idToken` i funktionen. Och gar det kanske att gora ett object med de lokala states som finns?
Att geoError och coordinates ligger i ett objekt.

Men det ar lite picky, du kan annars bara ignorera och merga